classes/class-wc-connect-nux.php

```diff
@@ -98,6 +98,37 @@ public function register_add_service_to_zone_pointer( $pointers ) {
 			return $pointers;
 		}
 
+		public static function get_banner_type_to_display( $status = array() ) {
```

Comment:
When taking in associative array parameters, we should check for keys existing or use a defaults mechanism to avoid PHP warnings for missing array keys.

Also, this function is defined as `static` but not called statically.